src/main/java/racingcar/domain/data/RoundCount.java

```diff
@@ -0,0 +1,27 @@
+package racingcar.domain.data;
+
+import racingcar.constants.Message;
+
+public class RoundCount {
+    private final int roundCount;
+
+    public RoundCount(String roundCount) {
+        this.roundCount = validateRoundCount(roundCount);
+    }
+
+    public int getRoundCount() {
+        return roundCount;
+    }
+
+    private int validateRoundCount(String roundCount) {
+        try {
+            int count = Integer.parseInt(roundCount);
+            if (count <= 0) {
+                throw new IllegalArgumentException(Message.ROUND_COUNT_AT_LEAST_ONE);
+            }
+            return count;
+        } catch (NumberFormatException e) {
+            throw new IllegalArgumentException(Message.ROUND_COUNT_NOT_NUMBER);
+        }
+    }
```

Comment:
이 부분은 메서드로 분리해 가독성을 개선할 수 있을 것 같아요!

```suggestion
    private int validateRoundCount(String roundCount) {
        int count = tryParseInt(roundCount);
        
        if (count <= 0) {
            throw new IllegalArgumentException(Message.ROUND_COUNT_AT_LEAST_ONE);
        }
    }

    private int tryParseInt(String roundCount) {
        try {
            int count = Integer.parseInt(roundCount);
            return count;
        } catch (NumberFormatException e) {
            throw new IllegalArgumentException(Message.ROUND_COUNT_NOT_NUMBER);
        }
    }
```

이런 느낌은 어떨까요?